src/arch/win/unix2nt_cc

```diff
@@ -178,52 +185,68 @@ do
 		;;
 	-E)
 		CL="$CL -E"
+		ML="$ML -EP"
 		;;
 	-O0)
 		CL_OPTS="$CL_OPTS -Od"
 		;;
 	-Og)
 		CL_OPTS="$CL_OPTS -DNDEBUG -Od"
+		ML_OPTS="$ML_OPTS -DNDEBUG"
 		;;
 	-O|-O1|-Os)
 		CL_OPTS="$CL_OPTS -DNDEBUG -O1"
+		ML_OPTS="$ML_OPTS -DNDEBUG"
 		;;
 	-O2)
 		CL_OPTS="$CL_OPTS -DNDEBUG -O2"
+		ML_OPTS="$ML_OPTS -DNDEBUG"
 		;;
 	-O3|-Ofast)
 		CL_OPTS="$CL_OPTS -DNDEBUG -Ox"
+		ML_OPTS="$ML_OPTS -DNDEBUG"
 		;;
 	-g)
-		CL_OPTS="$CL_OPTS -Z7"
+		CL_OPTS="$CL_OPTS -Z7 -GS-"
+		ML_OPTS="$ML_OPTS -Zi -Zd"
 		LINK_OPTS="$LINK_OPTS -DEBUG"
 		CL_MT='-MTd'
 		;;
 	-c) 
-		CL="$CL -c";;
+		CL="$CL -c"
+		ML="$ML -c"
+        ;;
 	-D)
 		CL="$CL -D$1"
+		ML="$ML -D$1"
 		shift
 		;;
 	-U)
 		CL="$CL -U$1"
+		ML="$ML -U$1"
 		shift
 		;;
 	-D*)
 		def=`echo $arg | awk -F@ '{print substr($1,3)}'`
 		CL="$CL -D$def"
+		ML="$ML -D$def"
 		;;
 	-U*)
 		def=`echo $arg | awk -F@ '{print substr($1,3)}'`
 		CL="$CL -U$def"
+		ML="$ML -U$def"
 		;;
 	-I)
-		CL="$CL -I`cygpath -w $1`"
+		path="`cygpath -w $1`"
+		CL="$CL -I$path"
+		ML="$CL -I$path"
```

Comment:
`CL`->`ML`. In bash, you can use `ML+=" -I..."` to avoid such an issue. (same below)